internal/kafka/handler.go

```diff
@@ -4,11 +4,13 @@ import (
 	"context"
 	"encoding/json"
 	"strings"
+	"time"
 
 	"github.com/confluentinc/confluent-kafka-go/kafka"
 	"gorm.io/gorm"
 
 	"github.com/redhatinsights/payload-tracker-go/internal/config"
+	"github.com/redhatinsights/payload-tracker-go/internal/endpoints"
```

Comment:
Importing the endpoints package into the kafka package doesn't feel right.  I think we mixing areas of the code that should not be mixed here.